src/Microsoft.IdentityModel.Protocols/Configuration/HttpDocumentRetriever.cs

```diff
@@ -75,6 +76,18 @@ public HttpDocumentRetriever(HttpClient httpClient)
         /// </summary>
         public bool RequireHttps { get; set; } = true;
 
+        /// <summary>
+        /// If set specifies the protocol version to use when sending HTTP requests.
```

Comment:
Can we update the doc comments here and for HttpVersionPolicy to explain the relationship between the two and the order of priority.